server/datastore/mysql/hosts.go

```diff
@@ -5270,6 +5270,24 @@ func updateHostIssuesFailingPolicies(ctx context.Context, tx sqlx.ExecerContext,
 		return nil
 	}
 
+	// For 1 host, we use a single statement to update the host_issues entry.
```

Comment:
But in load test and most customers we don't use async update of policies so it will almost always be `len(hostIDs) == 1`. Am confused as to why it failed with the MySQL stack error.

Reply:
It failed with stack error when moving 100K hosts from one team to another. In that case, we have to clear/update calculated failing policies counts for all those hosts.

Adding back code for `len(hostIDs) ==1` is just an optimization for the usual case.

Reply:
Ah sorry, I missed `updateHostIssuesFailingPolicies` being used on other places.